modules/storage/opensearch-serverless/deployspec.yaml

```diff
@@ -0,0 +1,24 @@
+publishGenericEnvVariables: true
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.105.0
+      - pip install -r requirements.txt
+    build:
+      commands:
```

Comment:
One config suggestion. By default, AOSS will scale up the serverless cluster by 10 OCUs for search and 10 OCUs for index that may add up quite a lot to the bill. We can put a cap on that via:
```
aws opensearchserverless update-account-settings \
    --capacity-limits '{ "maxIndexingCapacityInOCU": 8,"maxSearchCapacityInOCU": 9 }'
    ```

Reply:
Some info on that [here](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-scaling.html).

Reply:
Shouldn't that be configurable..i.e. cap it only if the customer wants to??

Reply:
Documented in the readme. Good point